packages/next/build/babel/loader/get-config.ts

```diff
@@ -6,6 +6,7 @@ import loadConfig from 'next/dist/compiled/babel/core-lib-config'
 
 import { NextBabelLoaderOptions, NextJsLoaderContext } from './types'
 import { consumeIterator } from './util'
+import { isPage } from '../../utils'
```

Comment:
Random question: should the new file be called `util` given the existing convention?

Reply:
Hey, I have not created this utils file. Its a very big existing file with many functions. Are you suggesting to change the name of existing file?

And also I searched in the codebase. some files are `util` and some are `utils`.